src/MLJ.jl

```diff
@@ -45,7 +46,8 @@ function MMI.update(model::EvoTypes, verbosity::Integer, fitresult, cache, A, y)
     grow_evotree!(fitresult, cache)
   end
 
-  report = (feature_importances=importance(fitresult, A.names),)
+  # report = (feature_importances=importance(fitresult, A.names),)
+  report = ()
```

Comment:
Shouldn't it return the feature names, same as in the above `fit` function? 
`report = (features=A.names,)`